style.css

```diff
@@ -0,0 +1,334 @@
+}
+
+.navigation-list__item {
+  text-decoration: none;
+}
+
+.page-header__item_mobile {
+  display: none;
+}
+
+.page-header__item_wide {
+  background-image: url(images/shop.jpg);
+  min-height: 100%;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.login {
+  color: rgb(11, 55, 136);
+  font-weight: bold;
+}
+
+.login:hover {
+  transform: scale(1.1);
+}
+
+.placeholder {
+  border-radius: 6px;
+  box-shadow: 2px 2px 2px black;
+  gap: 12px;
+  text-align: center;
+}
+
+.placeholder:hover {
+  transform: scale(1.1);
+}
+#home {
+  display: flex;
+  align-items: center;
+  padding-top: 0;
+  padding: 30px 20px;
+}
+
+#welcome {
+  font-size: 12vmin;
+  padding-left: 8rem;
+  padding-top: 4rem;
+  font-family: "Pacifico";
+}
+
+.product-list {
+  display: grid;
+  grid-gap: 25px;
+  grid-template-columns: 1fr 1fr 1fr;
+  padding-left: 0;
+  padding-right: 0;
+  padding: 30px 20px;
+}
+
+.product-list__item {
+  background-color: #e7e1db;
+  color: #000000;
+  box-shadow: 4px 8px 4px black;
+  transition: 0.5s;
+}
+
+.product-list__item:hover {
+  box-shadow: 12px 16px 8px black;
+}
+
+.product-list img {
+  max-width: 100%;
+}
+
+#mobile {
+  max-width: 100%;
+  margin-top: 5px;
+}
+
+.product-list {
+  padding: 0;
+  margin-bottom: 5px;
+}
+
+.white {
+  font-weight: bold;
+}
+
+.button-container {
+  display: flex;
+  padding: 0;
+  gap: 10px;
+}
+
+.button {
+  background-color: rgb(255, 255, 255);
+  border: none;
+  border-radius: 5px;
+  padding: 4px;
+  color: #031b1b;
+  cursor: pointer;
+}
+
+.button:hover {
+  background-color: #000000;
+  color: white;
+  border: solid 1px #074b4d;
+  transition: border 0.5s;
+}
+
+.fa-cart-shopping {
+  font-size: 1.5rem;
+  cursor: pointer;
+  color: rgb(255, 255, 255);
+}
+
+.fa-cart-shopping:hover {
+  transform: scale(1.2);
+}
+
+.fa-cart-plus {
+  font-size: 1.5rem;
+}
+
+.media {
+  color: #000000;
+  font-weight: bold;
+}
+
+text {
+  color: #000000;
+  font-weight: bold;
+}
+
+.order {
+  padding: 30px;
+}
+
+.contact-part {
+  background-color: rgb(216, 213, 213);
+}
+
+.contact_details {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.sidemenu {
+  font-size: 10px;
+  display: none;
+  list-style: none;
+  position: fixed;
+  bottom: 80px;
+  right: 18px;
+  z-index: 99;
+  border: none;
+  outline: none;
+  cursor: pointer;
+  padding: 0;
+}
+
+.sidemenu li {
+  padding-bottom: 10px;
+}
+
+.sidemenu i {
+  font-size: 25px;
+  opacity: 0.8;
+  color: black;
+}
+
+.sidemenu i:hover {
+  opacity: 1;
+}
+
+.sidemenu-item {
+  color: #000000;
+  padding-right: 5px;
+}
+
+#topButton {
+  position: flex;
+  display: none;
+  opacity: 0.8;
+  color: #ffffff;
+  bottom: 20px;
+  right: 30px;
+  z-index: 103;
+  border: none;
+  background-color: #0f1011;
+  cursor: pointer;
+  border-radius: 3px;
+  padding: 5px;
+}
+
+#topButton img {
+  vertical-align: top;
+  width: inherit;
+}
+
+#topButton:hover {
+  opacity: 1;
+}
+

```

Comment:
Check the login and sign up forms responsiveness. on small screen they don't show.